### ISSUE TITLE
Make a clear distinction between continuous and discrete time system dynamics

### ISSUE THREAD

**Issue by mzanon** [mzanon/thesis#5]
Use different names for the functions.

**Comment by rienq** [comment]
I usually use $\phi$ for the discrete and $f$ for the continuous, but I guess you just define a new command for both so you can change it later ;-)

**Comment by mzanon** [comment]
Good idea, I introduced the commands, but now I am wondering if I should even distinguish the exact discrete time system from the simulated one... moreover, f is also used for the cost funciton in static optimisation... maybe I make that one capital?
